### ISSUE TITLE
chance of radio jammer

### ISSUE THREAD

**Issue by treendy** [treendy/TRGM2.Malden#65]

**Comment by Xyaren** [comment]
In https://github.com/michail-nikolaev/task-force-arma-3-radio/wiki/API:-Variables

there are 




player setVariable ["tf_receivingDistanceMultiplicator", 2] A multiplier for increasing, or lowering the distance from transmitter to receiver (player).
player setVariable ["tf_sendingDistanceMultiplicator", 0.5] A multiplier for increasing or lowering the range of transmission.

Setting this to a low value would lead to a realistic simulation of an overloaded frequency.

**Comment by Xyaren** [comment]
This is how I imagine it:
Jammers can spawn at nearby radio towers or spawned mobile communication towers.
They will appear as an electronic box (Generator Model ?) next to the tower.
They can be "Deactivated" by a hold action or by destroying either the box or the tower.
The range of the jammer is defined by its type (Static / Mobile) and possibly the altitude of the tip of the tower.

**Comment by treendy** [comment]
Yes, thats the kind of idea... however, I looked at the TFAR variables a while ago, and they wasnt right for what we need (cant remember why now)

But Johnny Drama found a script that does effect your radio transmission, im yet to test it to see if it has  a nice distortion effect based on range etc....
